package.json

```diff
@@ -46,12 +47,15 @@
     "prettier": "^1.16.1"
   },
   "scripts": {
+    "applyIdsToChartDefinitions": "node scripts/chartsDefinitionsApplyUniqueId.js",
+    "prestart": "yarn applyIdsToChartDefinitions",
+    "prebuild": "yarn applyIdsToChartDefinitions",
+    "predeploy": "yarn run build",
```

Comment:
Keeping `pre` and `post` commands around their respective action e.g.
```js
    "prestart": ....
    "start": ... 
    "poststart" ...
```
makes it easy to track what's happening me thinks.